src/main/base/browserwindow.ts

```diff
@@ -428,8 +428,12 @@ export class BrowserWindow {
         }
         break;
       case "linux":
-        this.options.backgroundColor = "#1E1E1E";
         this.options.autoHideMenuBar = true;
+        if (!(utils.getStoreValue("visual.transparent") ?? false)) {
```

Comment:
I would say this is a little bit overcomplicated.

Reply:
D: I tried following the same logic as Windows for this part but if code duplication is an issue I could move the transparency option out of the platform switch statement and in its own block.

Or if you had a different idea on making it less complicated I don’t mind making the PR.

Thanks for merging nonetheless!